packages/@angular/cli/lib/config/schema.json

```diff
@@ -105,6 +105,10 @@
             "type": "string",
             "description": "The name of the TypeScript configuration file for unit tests."
           },
+          "karma": {
```

Comment:
Thoughts on using the `test` property that already exists?  It could be expanded to have an object form in additional to the string form.
That would allow the structure to mimic the top level `test` property and remove the need to add another property to an already long list.